### PR TITLE
feat: add actions menu to build list

### PR DESCRIPTION
**Pull request by JayCeeJr**
Add restart build button to build list 

Related Issue https://github.com/go-vela/community/issues/143
Screen Shot 2021-08-02 at 1 57 32 PM
Screen Shot 2021-08-02 at 1 27 11 PM

### CODE REVIEW

**Comment by JayCeeJr** [comment]
Since I was in this code, I added the name of the tag and the pull request number.

Screen Shot 2021-08-04 at 9 07 55 AM

**Review by wass3r — CHANGES_REQUESTED** [review]
thanks for the PR!

one initial ask: i would prefer if the addition for showing the tag (though appreciated) was a separate PR to keep the PR focused.

**Comment by JayCeeJr** [comment]
@wass3r  I reverted the tag and PR commit and opened a new PR

**Comment by wass3r** [comment]
one of the other PRs must have introduced a conflict 😿

**Comment by wass3r** [comment]
apologies for the delay. any chance we can address this visual discrepancy? (it's also visible in your screenshots)

actions

**Comment by JayCeeJr** [comment]
> apologies for the delay. any chance we can address this visual discrepancy? (it's also visible in your screenshots)
> 
> actions

I'm not sure how to accomplish keeping the space consistent, the menu is absolute positioned, and the rest is flex. The error adds an additional row which shifts the content up.

One solution would be to change the info box to `.build .info {justify-content: flex-start;}` (main.scss:517) however, i think that looks worse.
Screen Shot 2021-09-20 at 9 25 29 AM


Another would be to add `.build .info {min-height: 2em;}` (main.scss:532), which also doesn't look great.
Screen Shot 2021-09-20 at 9 31 13 AM

**Comment by wass3r** [comment]
what if we throw `position: relative` on the first row and move `details` in it? probably have to adjust a few things, like position on `details` element and such. haven't tried too much, but seems feasible from afar?

i can also take a crack at it after merge.

File: src/scss/style.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@charset 'utf-8';
+@charset "utf-8";
```
Comment (davidvader) [inline src/scss/style.scss:1]: was this change intentional?

Comment (JayCeeJr) [reply src/scss/style.scss:1]: Must have been the IDE trying to be helpful

**Review by wass3r — APPROVED** [review]
sorry for the delay but thanks for the work (again)!

tbh, not a huge fan of it being off-centered in the 2-row version, but it is consistent now 👍🏼

**Review by jbrockopp — APPROVED** [review]
LGTM
